krabs/krabs/ut.hpp

```diff
@@ -19,7 +19,13 @@ namespace krabs { namespace details {
     struct ut {
 
         typedef krabs::provider<> provider_type;
+        
+        struct filter_settings{
+        	std::vector<unsigned short> provider_filter_event_ids_;
```

Comment:
There are tabs here. Please use only spaces in all changes.

Reply:
Done